58/yashaslokesh/transcripts.py

```diff
@@ -0,0 +1,64 @@
+import requests
+import json
+import base64 as b64
+import sqlite3
+import os
+import itertools as it
+
+# Don't run this script too much or you get rate-limit blocked :(
+
+FIRST_SET_TOTAL_TRANSCRIPTS = 79
+SECOND_SET_START = 96
+SECOND_SET_END = 187
+
+"""
+    Generator function that will yield the transcript number and transcript for every
+    available transcript. This is done so that the strings don't take up too much memory
+    all at once and slow down the script (subsequent operation of adding text to sqlite)
+"""
+def transcripts(url):
+    for num in it.chain(range(0, FIRST_SET_TOTAL_TRANSCRIPTS + 1), 
+                        range(SECOND_SET_START, SECOND_SET_END + 1)):
+        # Creates file names going 000.txt, ...014.txt, ... 186.txt
+        transcript_num = f'{str(num).zfill(3)}.txt'
+        headers = {'Authorization': f'token {os.environ.get("TPT_OAUTH_TOKEN")}'}
+        result = requests.get(url + transcript_num, headers=headers)
+
+        # Obtain the textual portion of the file
+        data = json.loads(result.text)['content']
```

Comment:
```python
data = result.json()['content']
```

Reply:
That's right, I forgot a response object has the .json() method. Thanks

Reply:
@yashaslokesh 

No problem!

Reply:
Nice thing about `requests` indeed